install.sh

```
@@ -474,6 +474,55 @@
     return 1  # No preserved data
 }
 
+# Function to convert only relative paths to absolute paths
+convert_relative_to_absolute_path() {
+    local config_file=$1
+    local abs_path=$2
+    
+    # Look specifically for the audio export path in the export section
+    local export_section_line=$(grep -n "export:" "$config_file" | cut -d: -f1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)